src/helm/benchmark/static/utils.js

```diff
@@ -99,10 +99,10 @@ function renderPerturbationName(schema, perturbationName) {
   return schema.perturbationsField(perturbationName).display_name;
 }
 
-function renderScenarioDisplayName(scenario) {
+function renderScenarioDisplayName(scenario, scenario_spec) {
```

Comment:
Should we use camelCase here for variables, not properties?

Reply:
Fixed.